snuba/web/rpc/v1/resolvers/R_eap_items/storage_routing/routing_strategy_selector.py

```diff
@@ -82,21 +82,31 @@ def from_json(cls, config_json: str) -> "StorageRoutingConfig":
 
 
 class RoutingStrategySelector:
-    def get_storage_routing_config(self) -> StorageRoutingConfig:
+    def get_storage_routing_config(self, organization_id: int) -> StorageRoutingConfig:
```

Comment:
for future proofing I would just take the whole `in_msg` and make decisions there